2.3.0

HeavyTony2 opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->
Closes: #26 
Closes: #25

HeavyTony2 committed:
Update meta.yaml

HeavyTony2 commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.
I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * No valid build backend found for Python recipe for package `flask-caching` using `pip`. Python recipes using `pip` need to explicitly specify a build backend in the `host` section. If your recipe has built with only `pip` in the `host` section in the past, you likely should add `setuptools` to the `host` section of your recipe.

Commit:
MNT: Re-rendered with conda-build 24.9.0, conda-smithy 3.43.2, and conda-forge-pinning 2024.11.05.10.46.50

HeavyTony2 committed:
Pypi url issue

The "flask_caching-{{ version }}.tar.gz" was already included in the grayskull recipe.
It fixes the "RuntimeError: Could not download https://pypi.org/packages/source/F/Flask-Caching/Flask-Caching-2.3.0.tar.gz"  https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1072151&view=logs&j=7b6f2c87-f3a7-5133-8d84-7c03a75d9dfc&t=9eb77fd2-8ddd-5444-8fc0-71cb28dcb736&l=656

HeavyTony2 committed:
Update meta.yaml

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

HeavyTony2 commented:
@conda-forge-admin, please rerender

HeavyTony2 commented:
ping @ap--

Commit:
MNT: Re-rendered with conda-build 24.9.0, conda-smithy 3.44.3, and conda-forge-pinning 2024.11.12.14.24.54

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `host` section of the recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - For the `run` section of the recipe, you should usually use `python >={{ python_min }}` for the `python` entry.
   - For the `test.requires` section of the recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).

rxm7706 committed:
Add Python_Min, Remove markupsafe & werkzeug

https://github.com/pallets-eco/flask-caching/blob/v.2.3.0/setup.py

noarch: python recipes should usually follow the syntax in our documentation for specifying the Python version.

    For the host section of the recipe, you should usually use python {{ python_min }} for the python entry.
    For the run section of the recipe, you should usually use python >={{ python_min }} for the python entry.
    For the test.requires section of the recipe, you should usually use python {{ python_min }} for the python entry.
    If the package requires a newer Python version than the currently supported minimum version on conda-forge, you can override the python_min variable by adding a Jinja2 set statement at the top of your recipe (or using an equivalent context variable for v1 recipes).

HeavyTony2 committed:
Merge pull request #1 from rxm7706/main

Add Python_Min, Remove markupsafe & werkzeug

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

HeavyTony2 commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 24.9.0, conda-smithy 3.44.6, and conda-forge-pinning 2024.11.16.14.10.20

ap-- commented on recipe/meta.yaml:
```diff
@@ -1,13 +1,13 @@
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b7500c145135836a952e3de3a80881d9654e327a29c852c9265607f5c449235c
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/flask_caching-{{ version }}.tar.gz
```
Please make this small modification:

```suggestion
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_').lower() }}-{{ version }}.tar.gz
```

ap-- reviewed (changes requested):
Thank you for your contribution @HeavyTony2 and @rxm7706 

Please commit the small suggestion and then this is good to be merged.

HeavyTony2 committed:
Update recipe/meta.yaml

Co-authored-by: Andreas Poehlmann <ap--@users.noreply.github.com>
